projects/uitdatabank/docs/entry-api/events/status.md

```diff
@@ -1,8 +1,141 @@
-Coming soon.
-
-<!-- 
-  @todo 
-  Explain how to postpone/cancel an event, and optionally a reason.
-  Maybe show screenshot(s) of how postponed/cancelled events look on UiTinVlaanderen (as an example).
-  Also mention that a new event should be created for the new date(s) when postponed.
--->
+# Cancelling or postponing an event
+
+When [creating](./create.md) or (usually) when [updating](./update.md) your event, you can mark it as postponed or cancelled to indicate to attendees that it will be moved to another date or not take place at all.
+
+Cancelled events automatically get an *"(geannuleerd)"* label next to their calendar info in online calendars. For example, on UiTinVlaanderen:
```

Comment:
Maybe useful to link to the calendar summary guide (https://docs.publiq.be/docs/uitdatabank/search-api/calendar-summaries) here?

Reply:
Done!